### DISCUSSION TITLE
error 3 Bad request

### DISCUSSION THREAD

**Discussion by rantianhua** [socketio/socket.io#4467] [Q&A] [answered]
I always received this error message in my wss connection:

```
 {
    name: 'TRANSPORT_MISMATCH',
    transport: 'websocket',
    previousTransport: 'polling'
}
```

I want to know what's wrong with my connection and how can I solve it?

**Comment by gibloor** [comment, marked as answer]
It seems like you're encountering an issue where the frontend is likely using WebSocket, but when Nginx (or whatever you're using) forwards the request to the server, it switches the transport to polling. When the server tries to send a signal using WebSocket, this error occurs.

Configure NGINX!

**Reply by darrachequesne** [reply]
@gibloor thanks! I have added the list of error code in the troubleshooting guide: https://socket.io/docs/v4/troubleshooting-connection-issues/#troubleshooting-steps
